packages/fiori/src/ShellBarItem.ts

```diff
@@ -0,0 +1,86 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+
+/**
+ * @class
+ * The <code>ui5-shellbar-item</code> represents a custom item, that
+ * might be added to the <code>ui5-shellbar</code>.
+ * <br><br>
+ * <h3>ES6 Module Import</h3>
+ * <code>import "@ui5/webcomponents-fiori/dist/ShellBarItem";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.ShellBarItem
+ * @extends sap.ui.webc.base.UI5Element
+ * @abstract
+ * @tagname ui5-shellbar-item
+ * @implements sap.ui.webc.fiori.IShellBarItem
+ * @public
+ */
+
+ @customElement("ui5-shellbar-item")
+
+/**
+ * Fired, when the item is pressed.
+ *
+ * @event sap.ui.webc.fiori.ShellBarItem#click
+ * @allowPreventDefault
+ * @param {HTMLElement} targetRef DOM ref of the clicked element
+ * @event sap.ui.webc.fiori.ShellBarItem#click
```

Comment:
actually now there are two times `@event sap.ui.webc.fiori.ShellBarItem#click` - remove the second